src/components/footer.js

```diff
@@ -9,10 +9,18 @@ export default function Footer(options = {}) {
   return Component({
     render: function render() {
       let middleContent = '';
-      if (data.img) {
+      if (data.img && !data.url && !data.urlText) {
```

Comment:
This statement is the same as original could be removed

Reply:
Agreed